src/Google/Client.php

```diff
@@ -441,11 +440,16 @@ public function getAccessToken()
     return $this->token;
   }
 
+  /**
+   * @return string|null
+   */
   public function getRefreshToken()
   {
     if (isset($this->token['refresh_token'])) {
       return $this->token['refresh_token'];
     }
+
+    return null;
```

Comment:
What the point of this? By default methods return null if no value is returned

Reply:
This is about clean code mainly. An indication that the method returns null on purpose and not because someone forgot a statement. PhpStorm for example raises an issue when returns with and without a value are mixed.